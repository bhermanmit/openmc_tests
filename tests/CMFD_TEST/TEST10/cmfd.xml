<?xml version="1.0"?>
<cmfd>

  <mesh>
    <origin> -10 -10 -1 </origin>
    <dimension> 10 10 1 </dimension>
    <width> 2.0 2.0 2.0 </width>
    <albedo> 0.0 0.0 0.0 0.0 1.0 1.0 </albedo>
<!--    <map> 
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
           2 2 2 2 2 2 2 2 2 2
    </map>
-->
<!--    <energy>0.0 1.0 20.0</energy> -->
  </mesh>

</cmfd>
