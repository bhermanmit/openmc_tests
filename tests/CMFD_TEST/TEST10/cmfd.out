MESH (1. 1, 1)  GROUP 1           5.25308E-04
MESH (1. 1, 1)  GROUP 2           -5.01554E-04
MESH (2. 1, 1)  GROUP 1           4.74362E-04
MESH (2. 1, 1)  GROUP 2           -4.97227E-04
MESH (3. 1, 1)  GROUP 1           5.48165E-04
MESH (3. 1, 1)  GROUP 2           -5.22763E-04
MESH (4. 1, 1)  GROUP 1           6.32980E-04
MESH (4. 1, 1)  GROUP 2           -6.37947E-04
MESH (5. 1, 1)  GROUP 1           6.17556E-04
MESH (5. 1, 1)  GROUP 2           -6.20696E-04
MESH (6. 1, 1)  GROUP 1           6.41808E-04
MESH (6. 1, 1)  GROUP 2           -6.19537E-04
MESH (7. 1, 1)  GROUP 1           5.25992E-04
MESH (7. 1, 1)  GROUP 2           -5.36708E-04
MESH (8. 1, 1)  GROUP 1           5.39976E-04
MESH (8. 1, 1)  GROUP 2           -5.42427E-04
MESH (9. 1, 1)  GROUP 1           4.98369E-04
MESH (9. 1, 1)  GROUP 2           -4.99170E-04
MESH (10. 1, 1)  GROUP 1          5.14364E-04
MESH (10. 1, 1)  GROUP 2          -5.11658E-04
MESH (1. 2, 1)  GROUP 1           5.03665E-04
MESH (1. 2, 1)  GROUP 2           -4.82035E-04
MESH (2. 2, 1)  GROUP 1           3.86532E-04
MESH (2. 2, 1)  GROUP 2           -4.23638E-04
MESH (3. 2, 1)  GROUP 1           4.35932E-04
MESH (3. 2, 1)  GROUP 2           -4.78044E-04
MESH (4. 2, 1)  GROUP 1           3.16076E-04
MESH (4. 2, 1)  GROUP 2           -3.95957E-04
MESH (5. 2, 1)  GROUP 1           5.04761E-04
MESH (5. 2, 1)  GROUP 2           -4.84795E-04
MESH (6. 2, 1)  GROUP 1           5.05626E-04
MESH (6. 2, 1)  GROUP 2           -5.19310E-04
MESH (7. 2, 1)  GROUP 1           4.04322E-04
MESH (7. 2, 1)  GROUP 2           -4.71909E-04
MESH (8. 2, 1)  GROUP 1           4.73595E-04
MESH (8. 2, 1)  GROUP 2           -4.88128E-04
MESH (9. 2, 1)  GROUP 1           4.75759E-04
MESH (9. 2, 1)  GROUP 2           -4.30563E-04
MESH (10. 2, 1)  GROUP 1          4.24308E-04
MESH (10. 2, 1)  GROUP 2          -4.58147E-04
MESH (1. 3, 1)  GROUP 1           5.88787E-04
MESH (1. 3, 1)  GROUP 2           -5.90932E-04
MESH (2. 3, 1)  GROUP 1           4.28696E-04
MESH (2. 3, 1)  GROUP 2           -4.00243E-04
MESH (3. 3, 1)  GROUP 1           4.56521E-04
MESH (3. 3, 1)  GROUP 2           -4.22713E-04
MESH (4. 3, 1)  GROUP 1           4.83847E-04
MESH (4. 3, 1)  GROUP 2           -4.76906E-04
MESH (5. 3, 1)  GROUP 1           3.86005E-04
MESH (5. 3, 1)  GROUP 2           -4.13081E-04
MESH (6. 3, 1)  GROUP 1           5.06075E-04
MESH (6. 3, 1)  GROUP 2           -5.50003E-04
MESH (7. 3, 1)  GROUP 1           4.07353E-04
MESH (7. 3, 1)  GROUP 2           -4.37120E-04
MESH (8. 3, 1)  GROUP 1           4.39510E-04
MESH (8. 3, 1)  GROUP 2           -4.39788E-04
MESH (9. 3, 1)  GROUP 1           4.13221E-04
MESH (9. 3, 1)  GROUP 2           -4.03231E-04
MESH (10. 3, 1)  GROUP 1          6.02696E-04
MESH (10. 3, 1)  GROUP 2          -5.92890E-04
MESH (1. 4, 1)  GROUP 1           5.10313E-04
MESH (1. 4, 1)  GROUP 2           -5.31691E-04
MESH (2. 4, 1)  GROUP 1           4.80548E-04
MESH (2. 4, 1)  GROUP 2           -5.31806E-04
MESH (3. 4, 1)  GROUP 1           3.74446E-04
MESH (3. 4, 1)  GROUP 2           -3.50182E-04
MESH (4. 4, 1)  GROUP 1           4.64149E-04
MESH (4. 4, 1)  GROUP 2           -4.87029E-04
MESH (5. 4, 1)  GROUP 1           5.84906E-04
MESH (5. 4, 1)  GROUP 2           -5.91807E-04
MESH (6. 4, 1)  GROUP 1           5.17453E-04
MESH (6. 4, 1)  GROUP 2           -5.27655E-04
MESH (7. 4, 1)  GROUP 1           5.00805E-04
MESH (7. 4, 1)  GROUP 2           -4.96631E-04
MESH (8. 4, 1)  GROUP 1           5.58409E-04
MESH (8. 4, 1)  GROUP 2           -5.06777E-04
MESH (9. 4, 1)  GROUP 1           5.22578E-04
MESH (9. 4, 1)  GROUP 2           -4.97105E-04
MESH (10. 4, 1)  GROUP 1          6.34372E-04
MESH (10. 4, 1)  GROUP 2          -6.37963E-04
MESH (1. 5, 1)  GROUP 1           6.65385E-04
MESH (1. 5, 1)  GROUP 2           -6.27638E-04
MESH (2. 5, 1)  GROUP 1           4.81322E-04
MESH (2. 5, 1)  GROUP 2           -4.50980E-04
MESH (3. 5, 1)  GROUP 1           4.68708E-04
MESH (3. 5, 1)  GROUP 2           -4.91091E-04
MESH (4. 5, 1)  GROUP 1           4.40898E-04
MESH (4. 5, 1)  GROUP 2           -4.69612E-04
MESH (5. 5, 1)  GROUP 1           5.43623E-04
MESH (5. 5, 1)  GROUP 2           -5.17275E-04
MESH (6. 5, 1)  GROUP 1           5.42626E-04
MESH (6. 5, 1)  GROUP 2           -5.13589E-04
MESH (7. 5, 1)  GROUP 1           5.84792E-04
MESH (7. 5, 1)  GROUP 2           -5.43962E-04
MESH (8. 5, 1)  GROUP 1           4.41309E-04
MESH (8. 5, 1)  GROUP 2           -4.08010E-04
MESH (9. 5, 1)  GROUP 1           4.46323E-04
MESH (9. 5, 1)  GROUP 2           -4.22659E-04
MESH (10. 5, 1)  GROUP 1          6.11775E-04
MESH (10. 5, 1)  GROUP 2          -6.59537E-04
MESH (1. 6, 1)  GROUP 1           6.43837E-04
MESH (1. 6, 1)  GROUP 2           -6.05439E-04
MESH (2. 6, 1)  GROUP 1           4.08674E-04
MESH (2. 6, 1)  GROUP 2           -4.69999E-04
MESH (3. 6, 1)  GROUP 1           4.70501E-04
MESH (3. 6, 1)  GROUP 2           -4.94270E-04
MESH (4. 6, 1)  GROUP 1           5.47106E-04
MESH (4. 6, 1)  GROUP 2           -5.49972E-04
MESH (5. 6, 1)  GROUP 1           6.44008E-04
MESH (5. 6, 1)  GROUP 2           -5.74520E-04
MESH (6. 6, 1)  GROUP 1           5.00272E-04
MESH (6. 6, 1)  GROUP 2           -5.25549E-04
MESH (7. 6, 1)  GROUP 1           4.38970E-04
MESH (7. 6, 1)  GROUP 2           -4.44609E-04
MESH (8. 6, 1)  GROUP 1           5.46389E-04
MESH (8. 6, 1)  GROUP 2           -5.29990E-04
MESH (9. 6, 1)  GROUP 1           4.15944E-04
MESH (9. 6, 1)  GROUP 2           -4.64808E-04
MESH (10. 6, 1)  GROUP 1          6.56762E-04
MESH (10. 6, 1)  GROUP 2          -6.24633E-04
MESH (1. 7, 1)  GROUP 1           5.94450E-04
MESH (1. 7, 1)  GROUP 2           -5.80876E-04
MESH (2. 7, 1)  GROUP 1           4.87257E-04
MESH (2. 7, 1)  GROUP 2           -4.55930E-04
MESH (3. 7, 1)  GROUP 1           5.24203E-04
MESH (3. 7, 1)  GROUP 2           -4.97482E-04
MESH (4. 7, 1)  GROUP 1           4.65264E-04
MESH (4. 7, 1)  GROUP 2           -5.20805E-04
MESH (5. 7, 1)  GROUP 1           5.66036E-04
MESH (5. 7, 1)  GROUP 2           -5.30226E-04
MESH (6. 7, 1)  GROUP 1           5.61862E-04
MESH (6. 7, 1)  GROUP 2           -5.92850E-04
MESH (7. 7, 1)  GROUP 1           5.02555E-04
MESH (7. 7, 1)  GROUP 2           -4.76880E-04
MESH (8. 7, 1)  GROUP 1           4.83892E-04
MESH (8. 7, 1)  GROUP 2           -5.31453E-04
MESH (9. 7, 1)  GROUP 1           4.95718E-04
MESH (9. 7, 1)  GROUP 2           -4.96527E-04
MESH (10. 7, 1)  GROUP 1          6.32329E-04
MESH (10. 7, 1)  GROUP 2          -6.18601E-04
MESH (1. 8, 1)  GROUP 1           5.86715E-04
MESH (1. 8, 1)  GROUP 2           -5.58010E-04
MESH (2. 8, 1)  GROUP 1           4.10342E-04
MESH (2. 8, 1)  GROUP 2           -4.13217E-04
MESH (3. 8, 1)  GROUP 1           3.69526E-04
MESH (3. 8, 1)  GROUP 2           -4.13947E-04
MESH (4. 8, 1)  GROUP 1           5.38948E-04
MESH (4. 8, 1)  GROUP 2           -4.84013E-04
MESH (5. 8, 1)  GROUP 1           5.20939E-04
MESH (5. 8, 1)  GROUP 2           -4.60278E-04
MESH (6. 8, 1)  GROUP 1           5.08498E-04
MESH (6. 8, 1)  GROUP 2           -4.93210E-04
MESH (7. 8, 1)  GROUP 1           4.04861E-04
MESH (7. 8, 1)  GROUP 2           -4.25346E-04
MESH (8. 8, 1)  GROUP 1           4.96366E-04
MESH (8. 8, 1)  GROUP 2           -4.70334E-04
MESH (9. 8, 1)  GROUP 1           4.40867E-04
MESH (9. 8, 1)  GROUP 2           -4.44008E-04
MESH (10. 8, 1)  GROUP 1          5.78538E-04
MESH (10. 8, 1)  GROUP 2          -5.34706E-04
MESH (1. 9, 1)  GROUP 1           4.42263E-04
MESH (1. 9, 1)  GROUP 2           -4.85156E-04
MESH (2. 9, 1)  GROUP 1           4.03757E-04
MESH (2. 9, 1)  GROUP 2           -3.86535E-04
MESH (3. 9, 1)  GROUP 1           3.93111E-04
MESH (3. 9, 1)  GROUP 2           -4.16002E-04
MESH (4. 9, 1)  GROUP 1           4.03469E-04
MESH (4. 9, 1)  GROUP 2           -4.58633E-04
MESH (5. 9, 1)  GROUP 1           4.60479E-04
MESH (5. 9, 1)  GROUP 2           -5.03597E-04
MESH (6. 9, 1)  GROUP 1           4.44684E-04
MESH (6. 9, 1)  GROUP 2           -4.37118E-04
MESH (7. 9, 1)  GROUP 1           4.54020E-04
MESH (7. 9, 1)  GROUP 2           -4.48042E-04
MESH (8. 9, 1)  GROUP 1           4.33589E-04
MESH (8. 9, 1)  GROUP 2           -4.54409E-04
MESH (9. 9, 1)  GROUP 1           4.16618E-04
MESH (9. 9, 1)  GROUP 2           -4.03579E-04
MESH (10. 9, 1)  GROUP 1          4.77093E-04
MESH (10. 9, 1)  GROUP 2          -4.43912E-04
MESH (1. 10, 1)  GROUP 1          5.64898E-04
MESH (1. 10, 1)  GROUP 2          -5.20646E-04
MESH (2. 10, 1)  GROUP 1          5.46815E-04
MESH (2. 10, 1)  GROUP 2          -5.17979E-04
MESH (3. 10, 1)  GROUP 1          5.62414E-04
MESH (3. 10, 1)  GROUP 2          -5.50133E-04
MESH (4. 10, 1)  GROUP 1          5.74201E-04
MESH (4. 10, 1)  GROUP 2          -5.95301E-04
MESH (5. 10, 1)  GROUP 1          6.59830E-04
MESH (5. 10, 1)  GROUP 2          -6.49193E-04
MESH (6. 10, 1)  GROUP 1          5.87536E-04
MESH (6. 10, 1)  GROUP 2          -6.36601E-04
MESH (7. 10, 1)  GROUP 1          6.16158E-04
MESH (7. 10, 1)  GROUP 2          -6.17657E-04
MESH (8. 10, 1)  GROUP 1          5.98387E-04
MESH (8. 10, 1)  GROUP 2          -5.51877E-04
MESH (9. 10, 1)  GROUP 1          4.68837E-04
MESH (9. 10, 1)  GROUP 2          -4.90069E-04
MESH (10. 10, 1)  GROUP 1         4.75047E-04
MESH (10. 10, 1)  GROUP 2         -5.00686E-04
